src/main/groovy/com/ullink/NugetClear.groovy

```diff
@@ -0,0 +1,17 @@
+package com.ullink
+
+
+class NugetClear  extends BaseNuGet{
+    def all = false
+
+    NugetClear() {
+        super('locals -clear')
```

Comment:
As it's 2 separate arguments, pretty sure that should better be:

    super('locals')
    args '-clear'